frontend/src/scenes/settings/project/HeatmapsSettings.tsx

```diff
@@ -0,0 +1,40 @@
+import { LemonSwitch } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { teamLogic } from 'scenes/teamLogic'
+import { userLogic } from 'scenes/userLogic'
+
+export function HeatmapsSettings(): JSX.Element {
+    const { userLoading } = useValues(userLogic)
+    const { currentTeam } = useValues(teamLogic)
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const { reportHeatmapsToggled } = useActions(eventUsageLogic)
+
+    return (
+        <>
+            <p>
+                If you use our JavaScript libraries, we can capture general clicks, mouse movements, and scrolling to
+                create heatmaps. No additional events are created, and you can disable this at any time.
+                <br />
+                Whereas Autocapture creates events whenever it can uniquely identify an interacted element, heatmaps are
+                generated based on overall mouse or touch positions and are useful for understanding more general user
+                behavior across your site.
```

Comment:
Curious - how does this not make any new events? Does it require that autocapture be on to capture click info?

Reply:
https://github.com/PostHog/posthog-js/blob/main/src/posthog-core.ts#L774-L779

We hide the heatmap data on other events and extract them during ingestion